tests/pipelines/test_predict_step.py

```diff
@@ -1,12 +1,125 @@
+import os
+from pathlib import Path
+import pytest
+from pyspark.sql import SparkSession
+from sklearn.datasets import load_diabetes
+
+from mlflow.exceptions import MlflowException
+from mlflow.utils.file_utils import read_yaml
+from mlflow.pipelines.utils import _PIPELINE_CONFIG_FILE_NAME
+from mlflow.pipelines.steps.predict import PredictStep
+from mlflow.pipelines.steps.preprocessing import _CLEANED_OUTPUT_FILE_NAME
+
 # pylint: disable=unused-import
 from tests.pipelines.helper_functions import (
-    clear_custom_metrics_module_cache,
+    enter_test_pipeline_directory,
+    enter_pipeline_example_directory,
     tmp_pipeline_exec_path,
     tmp_pipeline_root_path,
     train_and_log_model,
 )  # pylint: enable=unused-import
 
 
-def test_always_pass(tmp_pipeline_root_path, tmp_pipeline_exec_path):
-    assert tmp_pipeline_root_path != tmp_pipeline_exec_path
-    assert True
+@pytest.fixture(scope="module", autouse=True)
```

Comment:
May be worthwhile to move this to helper_functions.py since the dev team will be wanting to make the other steps work with spark.

Reply:
I was considering this, but it seems like the existing pattern is to have each test file define its own spark pytest fixture. Maybe because it has `autouse` on or each test has different configurations. Maybe we can defer this to future spark efforts by the dev team.